UnitsNet.Tests/CustomCode/MassTests.cs

```diff
@@ -57,6 +57,12 @@ public class MassTests : MassTestsBase
 
         protected override double ShortHundredweightInOneKilogram => 0.022046226218487758;
 
+        protected override double EarthMassesInOneKilogram => 1.6744248350691500000000000E-25;
+
+        protected override double SolarMassesInOneKilogram => 5.0264643347223100000000000E-31;
```

Comment:
I find different numbers than you.

`5.02785e-31` at 
https://www.google.com/search?q=SolarMassesInOneKilogram&oq=SolarMassesInOneKilogram&aqs=chrome..69i57.295j0j9&sourceid=chrome&ie=UTF-8

`5.02785431e-31` at https://www.convertunits.com/from/kilograms/to/solar+masses
`5.0276521e-31` at http://astronomy.swin.edu.au/cosmos/S/Solar+Mass

I suppose this is another variable definition, but which is more correct?

Reply:
As for Earth Mass, there are multiple ways to calculate this value. I took the one from Wikipedia because it has more "precision" in the number, ie more decimals. But I don't mind taking another.

Reply:
The value I find the most as the Solar Mass is something around 1.989x10^30. Google also says that you divide the mass value by 1.989x10^30 to get the value of on solar mass in one kg. The difference is that I took a value of 1.98947x10^31 which has a slighty better precision. But at that scale one or the other is pretty much the same.